.github/workflows/cd.yml

```diff
@@ -0,0 +1,42 @@
+on:
+  push:
+  pull_request:
```

Comment:
Why do we want this triggered on pull requests?

Reply:
You probably don't. Sometimes I want to test building the image if changes happen. Let's remove it for now.